scripts/local-universe.py

```diff
@@ -45,9 +45,9 @@ def main():
     parser.add_argument(
         '--include',
         default='',
-        help='Command separated list of packages to include. If this option '
-        'is not specified then all packages are downloaded. E.g. '
-        '--include="marathon,chronos"')
+        help='Command (,) separated list of packages to include. For each '
+        'package specify both the name and version by separating them with a '
+        'colon (:). E.g. --include="marathon:1.4.2,chronos:2.5.0"')
```

Comment:
@justinrlee it's generally a good idea to use explicit versions, because it makes the build deterministic. It might seem like a benefit for a customer to get package updates automatically when they rebuild their local universe image, but I'd bet they'd prefer to know when the version is changing and what they need to do to handle upgrades safely.